lib/proxy.js

```diff
@@ -678,13 +682,27 @@ var respondNotFound = function (req, res) {
   res.end();
 };
 
+var onRequestCallback = function (p, req, res, options) { };
```

Comment:
I think these should not be globals, as if this module would be a singleton, it could imply serious side effects if the user create several instances of the proxy in a single server.